README.md

```diff
@@ -55,8 +51,8 @@ If you have obtained a private LH Cloud Sandbox, you can skip this step and just
 
 To run a LittleHorse Server locally in one command, you can run:
 
-```
-docker run --name littlehorse -d -p 2023:2023 -p 8080:8080 ghcr.io/littlehorse-enterprises/littlehorse/lh-standalone:0.12.1
+```sh
+docker run --name littlehorse -d -p 2023:2023 -p 8080:8080 ghcr.io/littlehorse-enterprises/littlehorse/lh-standalone:latest
 ```
 
 Using the local LittleHorse Server takes about 15-25 seconds to start up, but it does not require any further configuration. Please note that the `lh-standalone` docker image requires at least 1.5GB of memory to function properly. This is because it runs kafka, the LH Server, and the LH Dashboard (2 JVM's and a NextJS app) all in one container.
```

Comment:
"This is because it runs kafka, the LH Server, and the LH Dashboard (2 JVM's and a NextJS app) all in one container" needs to be 
This is because it runs Apache Kafka, the LH Server, and the LIttleHorse Dashboard all in one container
Don't use LH and you have to say "Apache Kafka"